app/serializers/decidim/participatory_documents/suggestion_serializer.rb

```diff
@@ -47,6 +48,16 @@ def submitted_on(suggestion)
       def suggestion_body(suggestion)
         truncate(translated_attribute(suggestion.body), length: 50)
       end
+
+      def answer_text(suggestion)
+        text_length = Decidim::ParticipatoryDocuments.max_export_text_length
+
+        if text_length.positive?
+          truncate(translated_attribute(suggestion.answer), length: text_length)
+        else
+          translated_attribute(suggestion.answer)
+        end
```

Comment:
```suggestion
        truncated_text(suggestion.answer)
      end
      
      def truncated_text(text)
        text_length = Decidim::ParticipatoryDocuments.max_export_text_length
        return text unless text_length.positive?
        
        truncate(translated_attribute(suggestion.answer), length: text_length)
        end
```

I'd apply this truncation in a more generic way and also apply it to the suggestion_body as well